### DISCUSSION TITLE
权限存储问题提问

### DISCUSSION THREAD

**Discussion by nizpxox** [zuihou/zuihou-admin-cloud#339] [02.提问]
我登录之后进入首页，我点基础平台切换回弹出4个接口请求，去获取我当前登录用户信息，以及资源权限。我想了解一下这个东西我点击对应菜单，这个权限校验是做了redis缓存么，还是就查这一遍mysql之后就
不需要后续得校验了，等我每一次在切换平台时候在获取一遍。![image](https://github.com/dromara/lamp-cloud/assets/141626156/415c4b18-e01a-40d6-b7c9-ec7ed6f9a4b7)

**Comment by zuihou** [comment]
前3个接口是，每次刷新页面，切换应用 都会重新请求后端，不刷新（F5）页面不会重复请求。

后端先读redis，后读数据库

**Reply by nizpxox** [reply]
这个就相当于我查出来之后缓存在redis里么，后续我点击其他菜单，就是通过redis去查权限，如果redis没有在查mysql是这个意思么

**Reply by zuihou** [reply]
是的
